### PR TITLE
Implement Spectrum Inquiry #4, #5

### PR DESCRIPTION
**Pull request by krlinus**

### CODE REVIEW

File: src/harness/testcases/spectrum_inquiry_testcase.py [hunk 1]
```diff
@@ -30,6 +30,105 @@ def tearDown(self):
...
+    # Register the device
+    device_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    self._sas_admin.InjectFccId({'fccId': device_a['fccId']})
+    # Locate this device close to coast (Half Moon Bay, Calif.)
+    device_a['installationParam']['latitude'] = 37.444267
+    device_a['installationParam']['longitude'] = -122.428628
+    request = {'registrationRequest': [device_a]}
+    response = self._sas.Registration(request)['registrationResponse'][0]
+    cbsd_id = response['cbsdId']
```
Comment (sushantparkhi) [inline src/harness/testcases/spectrum_inquiry_testcase.py]: Maybe a check for Registration ResponseCode
   ` self.assertEqual(response['response']['responseCode'], 0)
`

File: src/harness/testcases/spectrum_inquiry_testcase.py [hunk 2]
```diff
@@ -30,6 +30,105 @@ def tearDown(self):
...
+    # Register the device
+    device_b = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    self.assertEqual(device_b['cbsdCategory'], 'B')
+    self._sas_admin.InjectFccId({'fccId': device_b['fccId']})
+    # Locate this device close to coast (Half Moon Bay, Calif.)
+    device_b['installationParam']['latitude'] = 37.444267
+    device_b['installationParam']['longitude'] = -122.428628
+    request = {'registrationRequest': [device_b]}
+    response = self._sas.Registration(request)['registrationResponse'][0]
```
Comment (sushantparkhi) [inline src/harness/testcases/spectrum_inquiry_testcase.py]: Check for Registration ResponseCode
   ` self.assertEqual(response['response']['responseCode'], 0)
`

File: src/harness/testcases/spectrum_inquiry_testcase.py [hunk 3]
```diff
@@ -30,6 +30,105 @@ def tearDown(self):
...
+    }
+    request = {'spectrumInquiryRequest': [spectrum_inquiry_0]}
+    # Check spectrum inquiry response
+    response = self._sas.SpectrumInquiry(request)['spectrumInquiryResponse'][0]
+    self.assertEqual(response['cbsdId'], cbsd_id)
+    self.assertEqual(len(response['availableChannel']), 0)
+    self.assertEqual(response['response']['responseCode'], 0)
+    # Deregister the device and again register at new, non-coastal location
+    request = {'deregistrationRequest': [{'cbsdId': cbsd_id}]}
+    response = self._sas.Deregistration(request)['deregistrationResponse'][0]
```
Comment (sushantparkhi) [inline src/harness/testcases/spectrum_inquiry_testcase.py]: Check for Deregistration ResponseCode
   ` self.assertEqual(response['response']['responseCode'], 0)
`

File: src/harness/testcases/spectrum_inquiry_testcase.py [hunk 4]
```diff
@@ -30,6 +30,105 @@ def tearDown(self):
...
+    del request, response
+
+    # Query again, now in a location not affected by the coastal exclusion zone.
+    device_b = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    request = {'registrationRequest': [device_b]}
+    response = self._sas.Registration(request)['registrationResponse'][0]
```
Comment (sushantparkhi) [inline src/harness/testcases/spectrum_inquiry_testcase.py]: Check for Registration ResponseCode
   ` self.assertEqual(response['response']['responseCode'], 0)
`

Comment (krlinus) [inline src/harness/testcases/spectrum_inquiry_testcase.py]: Done

File: src/harness/testcases/spectrum_inquiry_testcase.py (hunk 3 above)
Comment (krlinus) [inline src/harness/testcases/spectrum_inquiry_testcase.py]: Done

File: src/harness/testcases/spectrum_inquiry_testcase.py (hunk 2 above)
Comment (krlinus) [inline src/harness/testcases/spectrum_inquiry_testcase.py]: Done

File: src/harness/testcases/spectrum_inquiry_testcase.py (hunk 1 above)
Comment (krlinus) [inline src/harness/testcases/spectrum_inquiry_testcase.py]: Done

File: src/harness/testcases/spectrum_inquiry_testcase.py (hunk 2 above)
Comment (janetliuboston) [inline src/harness/testcases/spectrum_inquiry_testcase.py]: Looks good.

**Comment by janetliuboston** [comment]
Looks good.

**Pull request merged**